scripts/report/_01_make_reports.py

```diff
@@ -752,10 +751,9 @@ def run_report_source(
                                      subject=subject, session=session))
 
         if condition in cfg.conditions:
-            title = f'Source: {config.sanitize_cond_name(condition)}'
+            title = f'Condition: {config.sanitize_cond_name(condition)}'
         else:  # It's a contrast of two conditions.
-            # XXX Will change once we process contrasts here too
-            continue
+            title = f'Contrast: {config.sanitize_cond_name(condition)}'
```

Comment:
Can you please revert this? We don't support contrasts in source space yet, and naming the source estimates `Condition` collides with our evoked data

Reply:
Is there something specific about contrasts and source space, beyond "just" using the appropriate combined evoked? If not, then I implemented _that_. Sorry if I'm getting this wrong.

Reply:
We discussed this in person and the gist is that we should address source level contrasts in a separate PR (in fact, there's already one)

Reply:
Thanks, reverted